src/wp-includes/class-wp-application-passwords.php

```diff
@@ -162,6 +171,28 @@ public static function get_user_application_password( $user_id, $uuid ) {
 		return null;
 	}
 
+	/**
+	 * Check if application name exists before for this user.
+	 *
+	 * @since 5.7.0
+	 *
+	 * @param int    $user_id User ID.
+	 * @param string $name    Application name.
+	 *
+	 * @return bool Provided application name exists or not.
+	 */
+	public static function user_application_name_exists( $user_id, $name ) {
```

Comment:
```suggestion
	public static function application_name_exists_for_user( $user_id, $name ) {
```

I think this reads more clearly. Though we could potentially drop the user bit entirely since all the methods on this class operate on a specific user.

Reply:
Agreed.